### PR TITLE
Include bid requests in critical path

### PR DESCRIPTION
**Pull request by warrengm**
Also includes some incidental formatting changes. I reverted most formatting diffs but left others in

### CODE REVIEW

File: lighthouse-plugin-ad-speed-insights/utils/graph.js [hunk 1]
```diff
@@ -14,8 +14,10 @@
 
 // @ts-ignore
 // eslint-disable-next-line
-const BaseNode = require('lighthouse/lighthouse-core/lib/dependency-graph/base-node');
+const BaseNode = require("lighthouse/lighthouse-core/lib/dependency-graph/base-node");
```
Comment (jburger424) [inline lighthouse-plugin-ad-speed-insights/utils/graph.js]: What does this accomplish?

File: yarn.lock [hunk 2]
```diff
@@ -145,11 +145,33 @@ argparse@^1.0.7:
   dependencies:
     sprintf-js "~1.0.2"
 
+array-differ@^2.0.3:
```
Comment (jburger424) [inline yarn.lock]: It doesn't look like you are using any new dependencies. Why the changes in yarn.lock without any in package.json?

File: lighthouse-plugin-ad-speed-insights/utils/graph.js (hunk 1 above)
Comment (warrengm) [reply lighthouse-plugin-ad-speed-insights/utils/graph.js]: I was trying an auto-formatter and accidentally made this change. Reverted

File: yarn.lock (hunk 2 above)
Comment (warrengm) [reply yarn.lock]: Reverted.
